oat-sa/tao-core-ui-fe
# feat/AUT-114/add new table type in toolbarPresets

Related to: https://oat-sa.atlassian.net/browse/AUT-114

Depends on:

- [ ] **update of "raphael" in separate task, here temporary updated to "2.2.7", to have ability install env**

**How to test:**
- go to item Authoring / Passage Authoring
- add A-block and table in it
- set cursor on table cell
- use the alignment buttons on the toolbar
- add new col or row and check that the new row or the new column will apply the same formatting 

![image](https://user-images.githubusercontent.com/25976342/148889590-837c10ed-9f6d-41b4-8f74-9c77e8a1f5e0.png)
